### PR TITLE
P#77250 Offer all estate status fields in the template

### PR DESCRIPTION
**Pull request by dai-eastgate**
related to issue https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/308

chaged logs:
-

### CODE REVIEW

**Review by LongTrong-exe — APPROVED** [review]

File: templates.dist/estate/default.php [hunk 1]
```diff
@@ -61,6 +61,15 @@
 	while ( $currentEstate = $pEstatesClone->estateIterator() ) :
 		$marketingStatus = $currentEstate['vermarktungsstatus'];
 		unset($currentEstate['vermarktungsstatus']);
+		unset($currentEstate['referenz']);
```
Comment (jmaas-onoffice) [inline templates.dist/estate/default.php:64]: I'm curious: Why are we unsetting the values here? Are they conflicting with something?

Comment (dai-eastgate) [reply templates.dist/estate/default.php:64]: They don't conflict. It's not in use at the moment so I just hid it
Do you want to display it or use it in any way?
image

Comment (jmaas-onoffice) [reply templates.dist/estate/default.php:64]: Ah, ok! I forgot that they get rendered.

There is also the option of adding those fields to the [`dontEcho` array](https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/324/files/79ce57d0bf507643e66912f02d29e1207a44ca16#diff-fe5dbe760501e4e626f9343c784248ce823bead31e4209da309b956045454e9cR30). ;)

Comment (dai-eastgate) [reply templates.dist/estate/default.php]: I added it you can review. :)

**Review by jmaas-onoffice — APPROVED** [review]

**Review by jmaas-onoffice — CHANGES_REQUESTED** [review]
image

**Review by jmaas-onoffice — APPROVED** [review]

**Comment by jmaas-onoffice** [comment]
I made a mistake with the requirements: This is backwards incompatible.

When users update, they would need to change their personalized templates so that the individual fields do not get output. We need to find a mechanism so that no template changes are needed on update.

Since this is quite a difficult thing to do properly, I will close this PR for now. Sorry for that!

**Pull request closed without merging**